src/python/Plane_TEST.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2021 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from ignition.math import AxisAlignedBox, Planed, Vector2d, Vector3d
+
+
+class TestPlane(unittest.TestCase):
+
+    def test_plane_constructor(self):
+        plane = Planed(Vector3d(1, 0, 0), 0.1)
+        self.assertEqual(plane.normal(), Vector3d(1, 0, 0))
+        # self.assertAlmostEqual(plane.offset(), 0.1, 1e-6)
```

Comment:
was there a problem with this expectation?

Reply:
enabled in https://github.com/ignitionrobotics/ign-math/pull/272/commits/ee6c81ba71f2c169cca23e10b6b960c82cc04259